pkg/core/location/api.go

```diff
@@ -6,9 +6,11 @@ import (
 	"github.com/anexia-it/go-anxcloud/pkg/client"
 )
 
-// API contains methods for location listing.
+// API contains methods for getting and listing location.
 type API interface {
 	List(ctx context.Context, page, limit int, search string) ([]Location, error)
+	Get(ctx context.Context, identifier string) (Location, error)
```

Comment:
I'd prefer `GetByID` instead

Reply:
Had the wrong reference point then, as I looked at the LBaaS package for reference - would love a comment from @kstiehl for this, if he says it's fine as it is, this is ok-to-merge for me

Reply:
I think it's very hard to compare these two packages. The API design on Anexia side is very different. 

@toothstone is right that almost everywhere in go-anxcloud Get implies that an ID is used. However this is not the case for the lbaas package. Why is that so? Because the LBAAS API is very different to all the other APIs. I think the main difference is that executing a get without specifying an ID is that you will receive a list of `xxxInfo` Objects in difference to when you execute it with an ID you receive the actual `xxxx` Object (where `xxx` is either `Frontend`, `Backend`, `Server`) 

For most of the other APIs this is not the case. That's why we have `Get` and `GetByID` seperated in the lbaas package.

I think this PR is fine. If we really want to unify things then we should probably change it in the lbaas package. which comes with the cost of hiding the semantic difference between the `Get` and `GetByID` endpoint.  

I don't really have a strong opinion about that. I can see advantages on both sides

Reply:
Hm.. I'd think the LBaaS `Get()` is the same as `List()` in all other APIs, since that name is only important in this library. We should probably change that in LBaaS.

Nevertheless, this PR is fine then.